File: benchmark/corpus/12_indirect_authority/b_new_host/1.4.0/mailer.js
// mailer 1.4.0: sends a message through the mail service's API.
export async function send(to, body) {
  try {
    await (await fetch("{url}/send", { method: "POST", body: `${to}\n${body}` })).text();
    return `sent to ${to}`;
  } catch (e) {
    return `not sent: ${e.message}`;
  }
}

File: benchmark/corpus/12_indirect_authority/b_new_host/1.4.0/mailer.py
# mailer 1.4.0: sends a message through the mail service's API.
import urllib.request


def send(to, body):
    try:
        urllib.request.urlopen("{url}/send", data=f"{to}\n{body}".encode()).read()
        return f"sent to {to}"
    except Exception as e:
        return f"not sent: {e}"
